src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1017,20 +1017,39 @@ Base.hcat(df1::AbstractDataFrame, df2::AbstractDataFrame, dfn::AbstractDataFrame
           makeunique=makeunique, copycols=copycols)
 
 """
-    vcat(dfs::AbstractDataFrame...)
+    vcat(dfs::AbstractDataFrame...; columns::Union{Symbol, AbstractVector{Symbol}}=:equal)
```

Comment:
Sorry, but since this PR was opened we realized we use `cols` rather than `columns` in general (e.g. in function names), so better call this `cols` I guess.

Reply:
changed to `cols` throughout

Reply:
I don't see it. Did you push the commit?

Reply:
fixed in documentation. I don't think there is anywhere else.